File: bot/commands/botAdmin/status.js
const ms = require("parse-ms")
const package = require("../../../package.json")
exports.run = async (bot) => {
  bot.registerCommand("status", async (message, args) => {
    bot.database.Userdata.find({}).toArray(async(err, docsUS) => {
      if (err) bot.log("error", err)

      bot.database.HuntBot.find({}).toArray(async(err, docsHB) => {
        if (err) bot.log("error", err)

        if (await bot.checkPermission(message, "botAdmin")) {
          const uptime = ms(bot.uptime)
          const statusEmbed = {
            embed: {
              title: `${bot.user.username} Status`,
              color: bot.getEmbedColor(bot, message),
              thumbnail: {
                url: bot.user.avatarURL
              },
              fields: [
                {
                  name: "Total Guilds",
                  value: `\`${bot.guilds.size}\``,
                  inline: true
                },
                {
                  name: "Total Users",
                  value: `\`${docsUS.length}\``,
                  inline: true
                },
                {
                  name: "Current HuntBot",
                  value: `\`${docsHB.length}\``,
                  inline: true
                },
                {
                  name: "Uptime",
                  value: `\`${uptime.days} days, ${uptime.hours}h ${uptime.minutes}m ${uptime.seconds}s\``,
                  inline: true
                },
                {
                  name: "Version",
                  value: `\`${package.version}\``,
                  inline: true
                }
              ],
              timestamp: new Date()
            }
          }
          bot.createMessage(message.channel.id, statusEmbed)
        }
      })
    })
  })
}

File: src/main/java/dastrangeboi/atlantis/Commands/Status.java
package dastrangeboi.atlantis.Commands;

import dastrangeboi.atlantis.Bot;
import dastrangeboi.atlantis.Database;
import net.dv8tion.jda.api.EmbedBuilder;
import net.dv8tion.jda.api.entities.Member;
import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;

import java.awt.*;

public class Status {

    private static Color getRoleColor(Member member) {
        return member.getColor();
    }

    private static String msToTime(long millis) {
        long seconds = millis / 1000;
        long minutes = seconds / 60;
        long hours = minutes / 60;
        long days = hours / 24;
        return days+" days, "+hours%24+"h, "+minutes%60+"m";
    }

    public static void run(GuildMessageReceivedEvent message) {
        int totalUsers = (int) Database.userdata.countDocuments();
        int allHuntbot = (int) Database.huntbot.countDocuments();
        String currentUptime = msToTime(System.currentTimeMillis() - Bot.startUpTime);

        EmbedBuilder statusEmbed = new EmbedBuilder()
                .setTitle("Atlantis Status")
                .setColor(getRoleColor(message.getGuild().getSelfMember()))
                .setThumbnail(Bot.bot.getSelfUser().getAvatarUrl())
                .addField("Total Guilds", "`"+Bot.bot.getGuilds().size()+"`", true)
                .addField("Total Users", "`"+totalUsers+"`", true)
                .addField("Current HuntBots", "`"+allHuntbot+"`", true)
                .addField("Uptime", "`"+currentUptime+"`", true)
                .addField("Version", "`"+Bot.version+"`", true);
        message.getChannel().sendMessage(statusEmbed.build()).queue();
    }
}
